### ISSUE TITLE
PMM2: doc issue on page  'HAProxy Instance Summary' (details/dashboards/dashboard-haproxy-instance-summary.html)

### ISSUE THREAD

**Issue by marcoshaw** [percona/pmm-doc#447]
Page appears emtpy.  Is there other pages for HAPROXY?

**Comment by PaulJacobs-percona** [comment]
No, not empty.  Take a look at the source if you're having trouble viewing it:
https://github.com/percona/pmm-doc/blob/main/docs/setting-up/client/haproxy.md

**Comment by PaulJacobs-percona** [comment]
Ah sorry, did you mean the dashboard appears empty? For that you'll be better off posting something on the forum:
https://forums.percona.com/c/percona-monitoring-and-management-pmm/percona-monitoring-and-management-pmm-v2/
